src/index.js

```diff
@@ -271,9 +271,23 @@ assign(Lrud.prototype, {
      */
     var handleOverride = function (overrideId) {
       var override = this.overrides[overrideId]
-      if (!override) return
+      if (!override) {
+        return
+      }
       if (override.id === id && key === override.direction) {
-        this._assignFocus(this._getActiveChild(node), override.target, 0, node.children.indexOf(activeChild), 0, event, this.nodes[override.target])
+        var offset = (override.direction === 'RIGHT' || override.direction === 'DOWN') ? 1 : -1
+
+        this._assignFocus(
+          this._getActiveChild(node), // activeChildId
+          override.target, // nextActiveChild (becomes the enter.id for the move event)
+          this.nodes[this.nodes[override.target].parent].children.indexOf(override.target), // nextActiveIndex
```

Comment:
This `this.nodes[this.nodes[override.target].parent].children.indexOf(override.target)` can become:
var targetNode =this.nodes[override.target]
var parentTargetNode = this.nodes[targetNode.parent]
var nextActiveIndex  = parentTargetNode.children.indexOf(override.target)

what do you think?

Reply:
Out of date now - handled via a base value